docs/reference-architectures/virtual-machines-windows/single-vm.md

```diff
@@ -187,38 +190,38 @@ For more information on deploying this sample reference architecture, visit our
 [disk-encryption]: /azure/security/azure-security-disk-encryption
 [enable-monitoring]: /azure/monitoring-and-diagnostics/insights-how-to-use-diagnostics
 [fqdn]: /azure/virtual-machines/virtual-machines-windows-portal-create-fqdn
-[github-folder]: http://github.com/mspnp/reference-architectures/tree/master/virtual-machines/single-vm
+[git]: https://github.com/mspnp/reference-architectures/tree/master/virtual-machines/single-vm
+[github-folder]: https://github.com/mspnp/reference-architectures/tree/master/virtual-machines/single-vm
 [group-policy]: https://technet.microsoft.com/en-us/library/dn595129.aspx
 [log-collector]: https://azure.microsoft.com/blog/simplifying-virtual-machine-troubleshooting-using-azure-log-collector/
 [manage-vm-availability]: /azure/virtual-machines/virtual-machines-windows-manage-availability
 [multi-vm]: multi-vm.md
-[naming conventions]: ../../best-practices/naming-conventions.md
+[naming-conventions]: /azure/architecture/best-practices/naming-conventions.md
```

Comment:
Links to our own docs should be a relative path to the MD file, so the validator bot can validate them.

Also for the '/azure/....' style of links, you would leave out the file suffix.

Reply:
OK, will fix